torch/distributed/rpc/api.py

```diff
@@ -250,7 +250,7 @@ def rpc_async(to, func, args=None, kwargs=None):
     Make a non-blocking RPC call to run function ``func`` on worker ``to``. RPC
     messages are sent and received in parallel to execution of Python code. This
     method is thread-safe. This method will immediately return a
-    ``torch.distributed.FutureMessage`` that can be awaited on.
+    Future<Message> that can be awaited on.
```

Comment:
It seems `Future<Message>` is still exposed as `torch.distributed.rpc.FutureMessage` instead of `Future<Message>`, and Python developers probably don't need to know about `Future<Message>`. So probably we still say `torch.distributed.rpc.FutureMessage` here?

Reply:
I will change it to expose Future<Message> then

Reply:
Ideally, we shouldn't expose `FutureMessage` or `Future<Message>` to python land. `Message` is an internal RPC concept that shouldn't leak to users. Can we just call this `Future` in python land?